v3/util/seed-etcd.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/bash -xe
+
+source /etc/environment
+docker pull behance/docker-aws-secrets-downloader:latest
+
+DL_TABLE="sudo docker run behance/docker-aws-secrets-downloader --table $SECRETS_TABLE"
+# Get all available secrets and configs
+AV_SECRETS=$($DL_TABLE --key secrets)
+AV_CONFIGS=$($DL_TABLE --key configs)
+
+echo "$AV_SECRETS" | while read line ; do
+    SECRET=$($DL_TABLE --key secrets --name $line)
+    SECRET_TYPE=`echo $SECRET | cut -d' ' -f2`
+    SECRET_PATH=`echo $SECRET | cut -d' ' -f3`
+    SECRET_VAL=`echo $SECRET | cut -d' ' -f4-`
+
+    if [[ "$SECRET_TYPE" != "etcd" ]]; then
+        continue
+    fi
+
+    etcdctl set -- $SECRET_PATH "$SECRET_VAL"
```

Comment:
So this part won't work for some of our etcd values. I copied Iz's method of writing the commands to a file and then executing it (see L27 in old setup-credentials) to work around this.